### PR TITLE
updated circleci_dicker_primary to circleci and updated to TF13

### PR DESCRIPTION
**Pull request by mdrummerboy09**
updated circleci_dicker_primary to circleci

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,12 +1,12 @@
 version: 2.1
 
 references:
-  circleci_docker_primary: &circleci_docker_primary trussworks/circleci-docker-primary:822fac1c30f3bb7d5d595bed5d2dc86265c4f2f0
+  circleci: &circleci trussworks/circleci:6986bb9022e5a83599feb66a7128a2d0fa12732a
```
Comment (ralren) [inline .circleci/config.yml:4]: I think you forgot a digit or two in the hash.

**Review by ralren — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (mdrummerboy09) [reply .circleci/config.yml:4]: hmm. I think its good I triple checked and there was no changes to the hash

Comment (ralren) [reply .circleci/config.yml:4]: alright. circleci is pissed about something and i assumed.

Comment (mdrummerboy09) [reply .circleci/config.yml:4]: its mad that I havent did the TF13 stuff

Comment (mdrummerboy09) [reply .circleci/config.yml:4]: totally worked when I updated everything to TF13

**Review by ralren — APPROVED** [review]

**Pull request merged**